akka-testkit/src/main/scala/akka/testkit/javadsl/TestKit.scala

```diff
@@ -63,11 +64,26 @@ class TestKit(system: ActorSystem) {
     }
   }
 
+  /**
+   * Return an `Infinite` duration of Scala's `Duration`.
+   */
+  def infiniteDuration: Infinite = Duration.Inf
```

Comment:
doesn't `Duration.Inf()` work for Java?
strange to add more api for scala Duration when we are deprecating it in javadsl

Reply:
LoL, yes.